scalafmt-cli/src/main/scala/org/scalafmt/cli/CliArgParser.scala

```diff
@@ -35,40 +35,55 @@ object CliArgParser {
     new scopt.OptionParser[CliOptions]("scalafmt") {
       override def showUsageOnError = false
 
-      def printAndExit(inludeUsage: Boolean)(ignore: Unit,
-                                             c: CliOptions): CliOptions = {
+      private def printAndExit(
+          inludeUsage: Boolean)(ignore: Unit, c: CliOptions): CliOptions = {
         if (inludeUsage) showUsage
         else showHeader
         sys.exit
         c
       }
 
-      def readConfigFromFile(file: String, c: CliOptions): CliOptions = {
+      private def readConfigFromFile(file: String, c: CliOptions): CliOptions = {
         readConfig(
           FileOps.readFile(
             AbsoluteFile.fromFile(new File(file), c.common.workingDirectory)),
           c
         )
       }
-      def readConfig(contents: String, c: CliOptions): CliOptions = {
+      private def readConfig(contents: String, c: CliOptions): CliOptions = {
         c.copy(config = Config.fromHoconString(contents).get)
       }
 
+      private def addFile(file: File, c: CliOptions): CliOptions = {
+        val absFile = AbsoluteFile.fromFile(file, c.common.workingDirectory)
+        c.copy(customFiles = c.customFiles :+ absFile)
+      }
+
       head("scalafmt", Versions.nightly)
       opt[Unit]('h', "help")
         .action(printAndExit(inludeUsage = true))
         .text("prints this usage text")
       opt[Unit]('v', "version")
         .action(printAndExit(inludeUsage = false))
         .text("print version ")
-      opt[Seq[File]]('f', "files")
```

Comment:
Would it be possible to keep this? We can make it hidden so that it won't appear in --help. Removing the option would break a lot of wrapper scripts, for limited benefits IMO.